op-node/rollup/types.go

```diff
@@ -322,6 +330,12 @@ func (cfg *Config) Check() error {
 	if err := checkFork(cfg.EcotoneTime, cfg.FjordTime, Ecotone, Fjord); err != nil {
 		return err
 	}
+	if err := checkFork(cfg.FjordTime, cfg.GraniteTime, Fjord, Granite); err != nil {
+		return err
+	}
+	if err := checkFork(cfg.GraniteTime, cfg.HoloceneTime, Granite, Holocene); err != nil {
```

Comment:
And holocene?

Reply:
Nothing comes after holocene. Or do you want to add interop? We didn't have anything after Fjord here so I didn't put anything after Holocene.